README.md

```diff
@@ -9,17 +9,14 @@ This project is based on Queso Queue Plus, which was originally developed by Sho
 
 ## How do I use this?
 
+You need to have docker engine and docker-compose installed on the system for this to work.  
+
 First, you must clone the project. Either download the source or run the following command:
 
 ```bash
-  git clone https://github.com/fluid-queue/fluid-queue
+  git clone https://github.com/liquidnya/fluid-queso-queue.git
 ```
 
-Next, install the dependencies for the project using the following command:
```

Comment:
Instead of removing the native instructions entirely, could we have a section for running natively and a section for Docker? Not everybody uses Docker, and just as some people would rather not have npm on their machines, some people would rather not have Docker on their machines.

Reply:
added in previous instructions for local install